scipy/linalg/flapack_other.pyf.src

```diff
@@ -42,15 +42,54 @@ subroutine <prefix2>gejsv(joba,jobu,jobv,jobr,jobt,jobp,m,n,a,lda,sva,u,ldu,v,ld
 
 end subroutine <prefix2>gejsv
 
+subroutine <prefix2>trexc(wantq,n,a,lda,q,ldq,ifst,ilst,work,info)
+    ! Reorder the Schur decomposition of a real matrix
+    ! using an orthogonal or unitary equivalence transformation.
+
+    callstatement { ifst++; ilst++; (*f2py_func)((wantq?"V":"N"),&n,a,&lda,q,&ldq,&ifst,&ilst,work,&info); }
+    callprotoargument char*,F_INT*,<ctype2>*,F_INT*,<ctype2>*,F_INT*,F_INT*,F_INT*,<ctype2>*,F_INT*
+
+    integer optional,intent(in),check(wantq==0||wantq==1) :: wantq=1
+    integer intent(hide),depend(a) :: n=shape(a,1)
+    <ftype2> intent(in,out,copy),dimension(lda,n) :: a
+    integer intent(hide),depend(a) :: lda=shape(a,0)
```

Comment:
all workspace vars need to be at least 1 so `MAX(1, shape(a,0))` same for ldq below

Reply:
In that case, shouldn't it be a `check()` instead?

Reply:
`a` can be empty but `lda` cannot be 0 even when `a` is empty.